injected/src/features/broker-protection/actions/click.js

```diff
@@ -51,6 +51,10 @@ export function click(action, userData, root = document) {
         const elements = getElements(rootElement, element.selector);
 
         if (!elements?.length) {
+            if (element.failSilently) {
```

Comment:
@brianhall - just a quick one on naming - when I saw ‘failSilently’ I was expecting something like ‘do nothing in this case’, like an empty `catch` block or something. 

But in both these cases, would `optional: true` be a better way to express it?

then it reads: `if no elements found, but this was optional anyway, return a success` and `if the element is disabled and it’s not optional, return an error`

Reply:
@shakyShane I like that naming better, however we’ve got failsilently in use [elsewhere](https://github.com/duckduckgo/content-scope-scripts/blob/c6b67bbe92b055ea0a44e8bdeca618d3e6627a19/injected/src/features/broker-protection/actions/expectation.js#L16), so probably best to standardize for now. Will definitely consider it for future refactors.